modular_skyrat/modules/modular_implants/code/nifs.dm

```diff
@@ -150,6 +143,10 @@
 	linked_mob.AddComponent(/datum/component/nif_examine)
 	RegisterSignal(linked_mob, COMSIG_LIVING_DEATH, PROC_REF(damage_on_death))
 
+	if(preinstalled_nifsofts)
+		send_message("Loading preinstalled NIFSofts, please wait...")
```

Comment:
It does. but there is a good reason for this. Due to the way that the modular persistence code is loaded in, if we load NIFSofts when the NIF is created, it will not have a reference for the `linked_mob` causing the NIFSoft to not work. 

With that aside, I don't really think that having it be that way is that jarring.  It's like starting your phone and having it tell you that it's installing updates/software.